web/config/webpack.config.js

```diff
@@ -574,6 +574,10 @@ module.exports = function (webpackEnv) {
       new InterpolateHtmlPlugin(HtmlWebpackPlugin, env.raw),
       new InterpolateHtmlPlugin(HtmlWebpackPlugin, {
         DESCRIPTION: appPackageJson.description,
+        AUTHOR: appPackageJson.author.name,
+        TITLE: appPackageJson.author.name,
+        TWITTER_SITE: appPackageJson.author.twitter,
+        META_NO_INDEX: env.raw.ENV !== 'production' ? `<meta name="robots" content="noindex" />` : '',
```

Comment:
En attendant que le site soit "ready", je ferais cela : 
```suggestion
        // TODO WEB : index production website when considered "ready" 
        META_NO_INDEX: '<meta name="robots" content="noindex" />',
        // META_NO_INDEX: env.raw.ENV !== 'production' ? `<meta name="robots" content="noindex" />` : '',
```